src/world.cpp

```diff
@@ -539,7 +541,7 @@ void World::do_tick() {
 			}
 			
 			// TODO: Higher literacy should mean lower births
-			ssize_t growth = pop->life_needs_met * (rand() % 10);
```

Comment:
this is intentional

Reply:
ssize_t is not standard and is not windows-defined